srcpkgs/amule/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'amule'
+pkgname=amule
+version=11066
+revision=1
+wrksrc=amule-project-amule-9fd038f
+build_style=gnu-configure
+configure_args="--enable-cas --enable-wxcas
+ --enable-amule-daemon --enable-amulecmd
+ --enable-amule-gui --enable-alc  --enable-alcc
+ --enable-webserver --disable-debug --enable-optimize
+ --enable-geoip --disable-upnp --enable-fileview
+ --with-boost  --with-wx-config=/usr/bin/wx-config-gtk3"
+hostmakedepends="tar autoconf automake gettext gettext-devel"
+makedepends="boost zlib-devel wxWidgets-gtk3-devel gd-devel geoip-devel crypto++-devel"
+depends="wxWidgets-gtk3 gd geoip crypto++"
+short_desc="Emule-like client for the eD2k and Kademlia networks"
+maintainer="hazayan <hazayan@hotmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/amule-project/amule"
+_commit="9fd038f8208f085dc7e0b52b73b2e18b6536437b"
+distfiles="https://github.com/amule-project/amule/tarball/${_commit}>${pkgname}-${version}.tar.gz"
+checksum="5f9149088630238f1282cc1b59cf235d32e030dbb8dfa9d12b863f4505ae8f54"
+
+pre_configure() {
+	./autogen.sh
+}
+
+do_install() {
+	make install DESTDIR="${DESTDIR}"
```

Comment:
this is [the default](https://github.com/void-linux/void-packages/blob/a3227ffb485c7c4c0993fd8929c0d48359482a81/common/build-style/gnu-configure.sh#L38)

Reply:
The build failed without that: no artifacts found in `destdir`.

Reply:
Without that, the build fails because of empty `destdir`

Reply:
Probably some other unrelated issue, removing `do_install` and using the default one should be exactly the same.

Reply:
Fixed.